src/main/java/controller/UserController.java

```diff
@@ -22,36 +22,38 @@
 public class UserController extends HttpServlet {
 
 	private static final long serialVersionUID = 1L;
-	private Logger logger = LogManager.getLogger(UserController.class);
-
-	UserRepository userRepository = new UserRepository();
+	private final Logger logger = LogManager.getLogger(UserController.class);
+	private final ObjectMapper mapper = new ObjectMapper();
+	private final UserService userService = new UserService();
 
 	protected void doGet(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
 		User user = null;
 		String id = request.getParameter("id");
 		try {
-			user = userRepository.getUser(id);
+			user = userService.get(id);
 		} catch (SQLException e) {
 			logger.error(e.toString());
+		} catch (ClassNotFoundException e) {
+			logger.error(e.toString());
 		}
-		String userData = new ObjectMapper().writeValueAsString(user);
+		String userData = mapper.writeValueAsString(user);
 		response.setContentType("application/json");
-		PrintWriter printWriter = response.getWriter();
-		printWriter.println(userData);
+		response.getWriter().println(userData);
 	}
 
 	protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
 		PrintWriter printWriter = response.getWriter();
 		response.setContentType("text/html");
```

Comment:
Wrong content type. Your API endpoints always return data as JSON. Change to `application/json` here and everywhere in the project.